src/gui/list_search/controls.rs

```diff
@@ -85,13 +133,21 @@ pub fn key_handler(
             Some(selected_command)
         }
         // Handle query input
-        Key::Backspace => {
+        // Backspace
+        KeyEvent {
+            code: KeyCode::Backspace,
+            modifiers: KeyModifiers::NONE,
+        } => {
             state.input.pop();
             apply_filter(state, namespace_tabs, trove_commands);
             None
         }
-        Key::Char(c) => {
-            state.input.push(c);
+        // c
+        KeyEvent {
+            code: KeyCode::Char('c'),
```

Comment:
This event should handle any character key press to refine the search filter of all commands, not just the single character 'c'
```
 KeyEvent {
            code: KeyCode::Char(c),
            modifiers: KeyModifiers::NONE,
        } => {
            state.input.push(c);
            apply_filter(state, namespace_tabs, trove_commands);
            None
        }
```

Reply:
Oops. I really messed up this one 😅. Fixed by last commit.